docs/notebooks/customize.ipynb

```diff
@@ -31,7 +31,7 @@
    "cell_type": "markdown",
```

Comment:
<p>This seems like it's wrong : Default values for <code>wmo</code> are <code>nm=11</code> and <code>nc=5</code>     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>